homework-2/src/app.js

```diff
@@ -0,0 +1,72 @@
+$(document).ready(function(){
+    $('.plans__slider').slick({
+        mobileFirst: true,
+        arrows: false,
+        dots: false,
+        slidesToShow: 1.2,
+        slidesToScroll: 1,
+        infinite: false,
+        autoplay: true,
+        autoplaySpeed: 3000,
+        variableWidth: true,
+        variableHeight: true,
+        responsive: [
+            {
+                breakpoint: 768,
+                setting: {
+                    slidesToShow: 4,
+                    slidesToScroll: 1,
+                }
+            }
+        ],
+    });
+    $('.problems__sliders').slick({
+        mobileFirst: true,
+        arrows: false,
+        dots: true,
+        swipe: false,
+        allowTouchMove: false,
+        slidesToShow: 1,
+        slidesToScroll: 1,
+    });
+    $('.three__sliders').slick({
+        mobileFirst: true,
+        arrows: false,
+        autoplay: true,
+        autoplaySpeed: 3000,
+        dots: true,
+        dotsClass: 'slick-dots slider__dots',
+        customPaging: function(slick, index) {
+          let image = $(slick.$slides[index]).find('.three__image').attr('src');
+          return '<img src="' + image + '" alt="" /> '
+        },
+        swipe: false,
+        allowTouchMove: false,
+    });
+});
+//-----------mobile
+const aroundBody = document.body;
+const headerX = document.querySelector('#headerX');
```

Comment:
for id it is better to use `document.getElementById`
also you have only one button for navigation menu but tons of constants for button and menu body
this implementation can be simplified
Also use css class instead of changing style by js as you made in `mobileMenu` function

Reply:
use document.getElementById and delete two const

Reply:
JS only slick slider